cli/command/formatter/image.go

```diff
@@ -235,6 +235,11 @@ func (c *imageContext) Digest() string {
 
 func (c *imageContext) CreatedSince() string {
 	createdAt := time.Unix(c.i.Created, 0)
+
+	if createdAt.IsZero() {
+		return "<none>"
```

Comment:
Thinking of a slightly different approach; I'll write down a comment / suggestion